doc/tutorial/index.rst

```diff
@@ -11,3 +11,4 @@ Tutorial
     repos
     queries
     transaction
+    force-arch
```

Comment:
It looks good, I am only thinking if it would be a bit clearer if it was added right after the `session`?
I like how the tutorials are ordered according to the workflow progression and adding this at the end disrupts that.

I had also one more idea that the page could be a child of the `session` page but I am not sure about that.

Reply:
Originally I was thinking about just adding the commented section to the `session` example - but given that this really is kind of a corner case, I didn't want to over-complicate the tutorial example.
But let me experiment with a session sub-page, it might work.

Reply:
I've changed the PR to make the forcearch tutorial a sub-section of session and I like it. What do you think?

Reply:
I like it as well. Thank you!